packages/xs-extra/xapi.master/opam

```diff
@@ -3,18 +3,27 @@ maintainer: "xen-api@lists.xen.org"
 build: [
 	["./configure"]
   [make]
-  [make "install"]
+]
+install: [
+    ["oasis" "setup"]
+    ["oasis" "setup.ml" "-install"]
 ]
 build-test: [make "test"]
-remove: ["rm" "%{bin}%/xapi"]
+remove: [
+    ["oasis" "setup"]
+    ["oasis" "setup.ml" "-uninstall"]
+    ["ocamlfind" "remove" "xapi"]
+    ["ocamlfind" "remove" "xapi-client"]
+    ["ocamlfind" "remove" "xapi-datamodel"]
+]
 depends: [
   "ocamlfind"
   "xapi-test-utils"
   "xapi-idl"
   "xapi-libs-transitional"
   "xen-api-client"
 	"xapi-netdev"
-  "omake"
+  "oasis"
```

Comment:
I think this and `ocamlfind` can be a build dependencies, like `"oasis" {build}`.